server/src/controllers/v1/cards.js

```diff
@@ -10,7 +10,13 @@ module.exports = {
       .then((card) => { res.status(201).send(card) })
       .catch((error) => { res.status(400).send(error) })
   },
-
+  destroy: (req, res) => {
+    Card
+      .findByPk(req.params.id)
+      .then((card) => { card.destroy() })
```

Comment:
`.then((card) => { card.destroy() })`

can be used without `{}` and `()` parentheses

`.then(card => card.destroy())`

Reply:
@datarockets/cuties thoughts about preventing `{}` and `()` parentheses usage where possible?

Reply:
@DDKatch I don't have sublime linter for that too 😄

Reply:
hah, in react part it'll be fixed automatically, but for backend part it doesn't work so 🤔 @AleksSenkou what do you think about PP for linters for everyone, in order to fix this ?

Reply:
@DDKatch what do you mean by `for backend part it doesn't work so`? Kind of

`.then(card => card.destroy())` will throw an error?

Reply:
> .then((card) => { card.destroy() })
> can be used without {} and () parentheses
> .then(card => card.destroy())

@AleksSenkou I mean my linter automatically fix this case for react part

Reply:
@DDKatch got it 👍 

PP works well for me. Any way we can firstly fo with infra task. @DDKatch can you please add one?

Reply:
@AleksSenkou yes, for sure